runtime/gc_base/StringTable.cpp

```diff
@@ -849,193 +859,4 @@ j9gc_internString(J9VMThread *vmThread, j9object_t sourceString)
 	return internedString;
 }
 
-j9object_t
-j9gc_allocStringWithSharedCharData(J9VMThread *vmThread, U_8 *data, UDATA length, UDATA resolveFlags) {
```

Comment:
@dmitripivkine Any objections to removing this?  I think it may date back to when we stored char[]s in the sharedclasses cache....

Reply:
@dmitripivkine any thoughts?

Reply:
remove it if there is no need any more